package.json

```diff
@@ -12,16 +12,17 @@
     "format:check": "prettier --check ."
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.9.1",
     "@prisma/client": "^6.0.0",
     "@radix-ui/react-avatar": "^1.1.1",
+    "@radix-ui/react-label": "^2.1.0",
     "@radix-ui/react-popover": "^1.1.2",
     "@radix-ui/react-scroll-area": "^1.2.1",
     "@radix-ui/react-separator": "^1.1.0",
-    "@hookform/resolvers": "^3.9.1",
-    "@radix-ui/react-label": "^2.1.0",
     "@radix-ui/react-slot": "^1.1.0",
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
+    "cmdk": "^1.0.0",
```

Comment:
what is this package for?

Reply:
Thanks to Shadcn, I have to delete dependencies manually, because Shadcn for no reason haven't got uninstall feature. So basically... I was testing something and forgot to delete shit from packege.json after deleting components